tests/testSdf.cpp

```diff
@@ -332,11 +332,11 @@ TEST(Joint, sdf_constructor_revolute) {
   EXPECT(assert_equal(screw_axis_j1_l1, j1->screwAxis(l1)));
 
   // Check transform from l0 com to l1 com at rest and at various angles.
-  Pose3 T_01comRest(Rot3::identity(), Point3(0, 0, 0.4));
+  Pose3 T_01com_rest(Rot3::identity(), Point3(0, 0, 0.4));
```

Comment:
rename all poses in this file to `toTfrom` ? E.g. `T_01com_rest` -> `com0Tcom1_rest`. I'm actually not sure about the comment in line 334. com0Tcom1_rest would be *from* com1 *to* com0, but the comment says different. If the comment *is* correct (?) should be `com1Tcom0_rest`

Reply:
#80 says we should name all the poses to `T_12`. I'll update the pose names here, but perhaps we should update that issue?

Reply:
Nope, you're right. We did agree on that rename and I forgot about it. Then just get rid of com? Still has to be `T_tf` in that order.

Reply:
And `M_tf` rather than `_rest`